README.md

```diff
@@ -63,4 +63,4 @@ point.
   pokyuser@3bbac563cacd:/workdir$
   ```
   At this point you should be able to follow the same instructions as described
-  in https://www.yoctoproject.org/docs/2.1/yocto-project-qs/yocto-project-qs.html#releases.
+  in https://www.yoctoproject.org/docs/2.2.1/yocto-project-qs/yocto-project-qs.html#releases.
```

Comment:
Thought you wanted to reference the most recent **released** documentation here: _latest_ links point to ongoing builds of documentation. They may reference, as is the case these days with 2.3, a yet unreleased version of the Yocto project. If we don't mind this could appear misleading to newcomers (people who probably could benefit most of this link indeed), I agree using _latest_ is definitely more self-maintaining.